docs/ga-requirements.md

```diff
@@ -0,0 +1,51 @@
+P0 (e.g. cannot GA without these):
+* End-to-end tests
+  * OTLP, Jaeger and Zipkin
+  * Tomcat, Wildfly, Spring Boot
+  * All Java LTS versions + latest Java version if non-LTS
+* Benchmarking
+  * Startup overhead benchmark
+  * Runtime overhead benchmark
+* Implement all required semantic attributes
+  * Non-required semantic attributes are nice to have, but not required for GA
+* Basic configuration points
+  * Add custom auto-instrumentation
+  * Ability to build "custom distro"
+* Documentation (tag "documentation" minus tag "contributor experience")
+  * All configuration options
+    * Standard OpenTelemetry SDK + Exporter configuration options
+    * Auto-instrumentation configuration options (e.g. disable/enable, peer.service mapping)
+  * For each instrumentation
+    * Document all spans that it captures
+      * Span names
+      * Span attributes (including explanation of any non-semantic attributes)
+      * Events
+    * Document any instrumentation-specific configuration
+    * Document any other effects (e.g. updating SERVER span name with route)
+  * Document the basic configuration points (above)
+    * How to write your own auto-instrumentation
+      * (much of this can be shared with contributor documentation below)
+    * How to build your own "custom distro"
+  * How to troubleshoot (start documenting common issues somewhere)
+* Library (manual) instrumentations corresponding to at least 20 auto-instrumentations, with shared code and tests
```

Comment:
I consider Brave to have very high instrumentation coverage, and it's a total of 25 (including some that are multi-version)
https://github.com/openzipkin/brave/tree/master/instrumentation

20 feels very high for a P0, my suggestion would be making this more use-case oriented.

    Libraries commonly used on Android
    Libraries commonly used with spring-boot

for example.